lib/in-process-server-test-helpers.js

```diff
@@ -31,13 +32,13 @@ function start() {
   }
   startCalled = true
 
-  const originalArgv = process.argv
-  // Modifying argv during import is a bit dirty, but it works, and avoids
-  // making bigger changes to server.js.
-  process.argv = ['', '', config.port, 'localhost']
+  // Modifying config is a bit dirty, but it works, and avoids making bigger
+  // changes to server.js.
+  serverConfig.bind = {
+    host: 'localhost',
+    port: config.port,
+  }
   const server = require('../server')
-
-  process.argv = originalArgv
```

Comment:
Oy. Since server-config was previously imported only from server.js, and now is being imported from base.js (albeit temporarily), it's now being imported much earlier. This change prevented an issue where the config is built using the arguments from `npm run test:js:server` instead of the overridden host and port. While this way of doing things is still pretty bad, it's slightly better than before.